docs/spec/examples.md

```diff
@@ -337,100 +246,27 @@ To detail the presence of the new `service_id`s and assign them to their applica
 
 ```csv
 service_id,monday,tuesday,wednesday,thursday,friday,saturday,sunday,start_date,end_date
-spring_train_latelong ,1,0,1,0,1,0,0,20240101,20240630
-spring_train_earlylong,0,1,0,1,0,0,0,20240101,20240630
-spring_train_earlylong,1,0,1,0,1,0,0,20240701,20241231
-spring_train_latelong ,0,1,0,1,0,0,0,20240701,20241231
-```
-
-### Jobs of entirely nonrevenue operations
-
-A special track inspection train is being operated on a particular day, supported by an extra crew position.
-
-_Note: The deadhead trips themselves could be defined in their own new `service_id` even without definining corresponding run data in `run_events.txt` using the `calendar_supplement.txt` or `calendar_dates_supplement.txt` files._
-
-#### `trips_supplement.txt`
-
-```csv
-route_id,service_id,trip_id,TODS_trip_type,direction_id
-line1,inspection_train,inspection_line1_ob,deadhead,0
-line1,inspection_train,inspection_line1_ib,deadhead,1
-line1,inspection_train,inspection_line2_ob,deadhead,0
-line1,inspection_train,inspection_line2_ib,deadhead,1
-line1,inspection_train,inspection_line3_ob,deadhead,0
-line1,inspection_train,inspection_line3_ib,deadhead,1
-```
-
-#### `stop_times_supplement.txt`
-
-```csv
-trip_id,arrival_time,stop_id
-inspection_line1_ob,downtown,08:00:00
-inspection_line1_ob,anytown,09:00:00
-inspection_line1_ib,anytown,09:15:00
-inspection_line1_ib,downtown,10:15:00
-inspection_line2_ob,downtown,11:00:00
-inspection_line2_ob,busyville,12:30:00
-inspection_line2_ib,busyville,13:00:00
-inspection_line2_ib,downtown,14:30:00
-inspection_line3_ob,downtown,15:00:00
-inspection_line3_ob,centerton,15:45:00
-inspection_line3_ib,centerton,16:00:00
-inspection_line3_ib,downtown,16:45:00
-```
-
-#### `calendar_dates_supplement.txt`
-
-```csv
-service_id,date,exception_type
-inspection_train,20240901
```

Comment:
I do like the idea of including this as an example, because I think this is a big use case for the distinct TODS-specific modeling of non-public trips. That said, we could certainly cut this down to two trips of two stop_times each to simplify things. But open to cutting if the rest of the working group believes it to not be worthwhile.